docker/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM alpine:3.8
```

Comment:
Can we create a subdirectory named `crux` - it seems strange to have a `quorum-crux` subdirectory, but not a crux one for the Crux only dockerfiles